website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown

```diff
@@ -63,11 +63,9 @@ resource "azurerm_orchestrated_virtual_machine_scale_set" "example" {
 
 * `capacity_reservation_group_id` - (Optional) Specifies the ID of the Capacity Reservation Group which the Virtual Machine Scale Set should be allocated to. Changing this forces a new resource to be created.
 
--> **NOTE:** `capacity_reservation_group_id` cannot be used with `proximity_placement_group_id`
+-> **NOTE:** The `capacity_reservation_group_id` cannot be used with `proximity_placement_group_id`
```

Comment:
Rather than adding `the` here, this probably would make more sense as:

```suggestion
-> **NOTE:** `capacity_reservation_group_id` cannot be specified with `proximity_placement_group_id`
```